src/style.css

```diff
@@ -0,0 +1,145 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  font-family: Anenir-Book, sans-serif;
+  font-weight: 300px;
+  font-size: 16px;
+  height: 100vh;
+  background-color: #fff;
+  /* padding: 71px 81px 71px 48px; */
+  margin: 0 auto;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  display: block;
+  position: relative;
+  margin: 0 auto;
+  margin-left: 48px;
+
+}
+
+.formBig {
+  margin-top: 71px;
+}
+
+.labelBig {
+  display: flex;
+  width: 900px;
+  height: 70px;
+  margin-bottom: 20px;
+}
+
+.labelSmall {
+  display: flex;
+  width: 427px;
+  height: 42px;
+  position: absolute;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px rgba(61, 78, 97, 0.1);
+}
+
+.smallInputCl {
+  display: block;
+  height: 42px;
+  padding: 12px 227px 10px 33px;
+
+  font-family: Avenir-Book, sans-serif;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.smallInputCl:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.smallInputCl:focus {
+  font-family: Anenir-Heavy, sans-serif;
+  font-weight: 900;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.bigInputCl {
+  display: block;
+  height: 70px;
+  padding: 25px 709px 23px 62px;
+
+  font-family: Anenir-Book, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.bigInputCl:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.bigInputCl:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-sizing: border-box;
+  border-radius: 3px;
+
+  font-family: Anenir-Heavy, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.btnSmall {
+  background: url("images/Search.svg") no-repeat scroll 0 0 transparent;
+  background-size: 11px;
+  border: none;
+  cursor: pointer;
+  font-weight: bold;
+  height: 11px;
+  padding: 0;
+  margin: 0;
+  width: 11px;
+  z-index: 1;
+  position: absolute;
+  left: 13px;
+  top: 15px;
+}
+
+.btnBig {
+  background: url("images/Search.svg") no-repeat scroll 0 0 transparent;
+  background-size: 19px;
+  border: none;
+  cursor: pointer;
+  font-weight: bold;
+  height: 19px;
+  padding: 0;
+  margin: 0;
+  width: 19px;
+  z-index: 1;
+  position: absolute;
+  left: 26px;
+  top: 25px;
+}
```

Comment:
You repeat a lot of styles, you need to have one class with similar styles and two with different ones